pyproject.toml

```diff
@@ -40,7 +40,7 @@ pyjexl = "^0.3.0"
 python-memcached = "^1.62"
 requests = "^2.31.0"
 urllib3 = ">=1.26.8,<3.0.0"
-uWSGI = "^2.0.26"
+gunicorn = "^23.0.0"
```

Comment:
The dependencies should be in alphabetical order (expect for `python`).

Reply:
done